presentation/src/main/java/com/knocklock/presentation/lockscreen/LockScreenUiState.kt

```diff
@@ -0,0 +1,10 @@
+package com.knocklock.presentation.lockscreen
+
+/**
+ * @Created by 김현국 2022/12/07
+ */
+
+sealed class NotificationUiState {
+    object Init : NotificationUiState()
+    data class Success(val notificationList: List<Notification>) : NotificationUiState()
```

Comment:
```suggestion
    object Empty: NotificationUiState()
    data class Success(val notificationList: List<Notification>) : NotificationUiState()
```

init 보다는 Empty로 rename하는건 어떤가요? 😄

Reply:
@jaeryo2357 좋습니당